src/services/analytics.service.js

```diff
@@ -13,15 +14,19 @@ import electronStore from './electron-store.service';
 export const MIXPANEL_KEY = '5840a1a4b9bdcdb518471f0e0830baa2';
 const DISTINCT_ID_KEY = 'distinct-id';
 
-type Event =
+export type EventType =
```

Comment:
It feels weird duplicating those strings here. It means that every time we want to track a new action we need to modify not only the saga but the logger as well

Reply:
mhmm! I think that's a feature, not a bug haha. 

If the only way to log an event was through the saga, it probably would feel redundant, but since components can log events as well, this is necessary to ensure that we aren't logging typo event names, or stuff we don't explicitly want to track.

It would be simpler if we added a rule that said only the saga can log analytics events, but then we'd be dispatching redux actions for no purpose other than for its side-effect, which doesn't sit well with me.